### ISSUE TITLE
live data

### ISSUE THREAD

**Issue by Argonghost** [commaai/laika#156]
Hi,

Im having trouble finding which file im supposed to modify, if i want to run laika on real time data captured by a software defined radio.  Any help on this? 
If that is possible, what are the laika requirements in order to run on real time SDR data from an SDR front end connected to host PC via USB.  Im using the ADALM Pluto SDR which has a python host API (PyADI) .... so in theory there must be a way to run laika right????

**Comment by haraschax** [comment]
Yes laika can do this, you just have to use laika to process the data.
